www/js/survey/enketo-survey.js

```diff
@@ -17,7 +17,11 @@ angular.module('emission.survey.enketo.launch', [
       $state.go("root.main.diary")
     });
   } else {
-    EnketoSurvey.init($stateParams.form_location, $stateParams.opts)
+    EnketoSurvey.init({
+      form_location: $stateParams.form_location,
+      opts: $stateParams.opts,
```

Comment:
just to confirm, these `opts` are currently unused, right?

Reply:
`opts` is use to pass the `data_key` selector in case we have multiple surveys or data sources.

Reply:
```js
      $state.go("root.main.enketosurvey", {
      form_location: "json/trip-end-survey_v9.json",
          opts: JSON.stringify({
            session: {
              data_key: 'manual/confirm_survey',
            }
          }),
      });
```